### PR TITLE
Adding all changes

### PR DESCRIPTION
**Pull request by DanyalNazir10**
Made a new repo and adding all the changes i have made until now.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+    <Header></Header>
```
Comment (Hassan-mergestack) [inline src/App.js:8]: replace this with <Headers/>

Comment (Hassan-mergestack) [reply src/App.js:8]: `<Headers/>`

File: src/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+                      </span>
+                      MY ACCOUNT
+                    </a>
+                    <ul
+                      className="dropdown-menu bg-black"
+                      aria-labelledby="myAccountDropdown"
+                    >
+                      <li>
+                        <a className="dropdown-item" href="#">
+                          Purchase History
```
Comment (Hassan-mergestack) [inline src/Header.jsx:127]: create an array inside lib folder, declare all button names there and use map that returns `<a/>` tag

File: src/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+                      >
+                        SHOP
+                      </a>
+                      <ul
+                        className="dropdown-menu"
+                        aria-labelledby="shopDropdown"
+                      >
+                        <li>
+                          <a className="dropdown-item" href="#">
+                            ALL PRODUCTS
```
Comment (Hassan-mergestack) [inline src/Header.jsx:244]: create an array inside lib folder, declare all button names there and use map that returns <a/> tag

File: src/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+                      >
+                        #BIOTRUST
+                      </a>
+                      <ul
+                        className="dropdown-menu"
+                        aria-labelledby="bioTrustTagDropdown"
+                      >
+                        <li>
+                          <a className="dropdown-item" href="#">
+                            OUR STORY
```
Comment (Hassan-mergestack) [inline src/Header.jsx:314]: create an array inside lib folder, declare all button names there and use map that returns <a/> tag

File: src/MainSection.jsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
+export default function MainSection() {
```
Comment (Hassan-mergestack) [inline src/MainSection.jsx:1]: we follow camel case it should be **mainSection**

File: src/MainSection.jsx [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+                <div
+                  className="col-lg-8"
+                  style={{ background: "brown", height: "700px" }}
+                >
+                  <div className="row" style={{ background: "white" }}>
+                    AGELESS MULTI-COLLAGEN® PROTEIN POWDER
+                  </div>
+                  <div className="row">
+                    <span class="stars">
+                    <i className="fa-solid fa-star" style={{color: "#3ed016;"}}></i>
```
Comment (Hassan-mergestack) [inline src/MainSection.jsx:49]: you can use loop here to iterate stars

File: src/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,389 @@
+import React from "react";
+
+export default function Header() {
```
Comment (Hassan-mergestack) [inline src/Header.jsx:3]: camelCase needs to be followed

File: src/App.js [hunk 8]
```diff
@@ -1,25 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import Header from "./Header";
+import MainSection from "./MainSection";
```
Comment (JavedJutt) [inline src/App.js:3]: use absolute path

File: src/Header.jsx (hunk 7 above)
Comment (JavedJutt) [reply src/Header.jsx:3]: can you create Header/index.jsx folder?

File: src/Header.jsx [hunk 9]
```diff
@@ -0,0 +1,389 @@
...
+                      </li>
+                      <li>
+                        <a className="dropdown-item" href="#">
+                          Customer Support
+                        </a>
+                      </li>
+                      <li>
+                        <a className="dropdown-item" href="#">
+                          Login
+                        </a>
```
Comment (JavedJutt) [inline src/Header.jsx:158]: use map here

File: src/Header.jsx [hunk 10]
```diff
@@ -0,0 +1,389 @@
...
+                        <li>
+                          <a className="dropdown-item" href="#">
+                            LIFESTYLE
+                          </a>
+                        </li>
+                        <li>
+                          <a className="dropdown-item" href="#">
+                            VIDEOS
+                          </a>
+                        </li>
```
Comment (JavedJutt) [inline src/Header.jsx:356]: ditto for map

File: src/MainSection.jsx [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+                  className="col-lg-4"
+                  style={{ background: "purple", height: "700px" }}
+                >
+                  <div className="vertical-slick-nav">
+                    <div>your content</div>
+                    <div>your content</div>
+                    <div>your content</div>
+                    <div>your content</div>
+                    <div>your content</div>
+                    <div>your content</div>
```
Comment (JavedJutt) [inline src/MainSection.jsx:37]: again map

File: src/index.js [hunk 12]
```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+import './styles/SCSS/main.css';
 import App from './App';
```
Comment (JavedJutt) [inline src/index.js:4]: absloute paths

File: src/styles/SCSS/main.css [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+  transform: translateY(-50%);
+  right: 0;
+}
+header .nav-divider {
+  border-top: 5px solid #33659e;
+  opacity: 1;
+}
+header .dropdown:hover .dropdown-menu {
+  display: block !important;
+}/*# sourceMappingURL=main.css.map */
```
Comment (JavedJutt) [inline src/styles/SCSS/main.css:42]: add eof here

**Pull request merged**